docker/cdash.docker

```diff
@@ -112,6 +112,9 @@ else                                                                   \
   php artisan dependencies:update;                                     \
 fi
 
+# Disable git repo ownership check system wide
+RUN git config --system --add safe.directory '*'
```

Comment:
suggestion: move this line up above line 75 (`USER www-data`) to resolve the permissions problem we encountered in CI.